pkg/util/taskgroup/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Peter Kristensen
```

Comment:
What's the original source for the included code?

Reply:
Originally taken from https://github.com/ptxmac/multierrgroup, added the comment with it.